### PR TITLE
Alerting with Loki tutorial

### PR DESCRIPTION
**Pull request by mellieA**
**What is this feature?**

Tutorial describing how to create alert notifications from logs

**Why do we need this feature?**

Help the community learn how to create notifications with loki & grafana alerting

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/tutorials/create-alerts-with-logs/index.md [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+
+1. Expression B shows a table of labels and values returned. The message label captured the message string from the log line
+ and the value shows the number of times that string occured during the evaluation interval.
+
+    |   labels         |       values              |
+    | ---------------- | --------------------------|
+    |  message=level=info method=GET url=/ status=500 | 27 |
+    |  message=level=info method=POST url=/ status=500 | 1 |
+
+1. Configure your alert evaluation behavior
```
Comment (Eve832) [inline docs/sources/tutorials/create-alerts-with-logs/index.md:94]:
```suggestion
1. Configure your alert evaluation behavior.
```

File: docs/sources/tutorials/create-alerts-with-logs/index.md [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+    * In the blank `text` box paste ```{{ $labels.message }}  has returned an error status {{$values.B}} times.```
+
+1. Click the **Save and exit** button at the top of the alert definition page
+
+### Create a Loki managed alert
+
+[Loki managed alerts](https://grafana.com/docs/loki/latest/rules/#alerting-and-recording-rules) are stored and evaluated by Loki. They use LogQL for their expressions. 
+
+1. Choose  Mimir or Loki managed alert to create an alert using Loki
+1. Select your Loki datasource from the drop-down
```
Comment (Eve832) [inline docs/sources/tutorials/create-alerts-with-logs/index.md:112]:
```suggestion
1. Select your Loki data source from the drop-down
```

File: docs/sources/tutorials/create-alerts-with-logs/index.md [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+        {{ end }}
+        {{ if gt (len .Alerts.Resolved) 0 }}
+            {{ len .Alerts.Resolved }} resolved:
+            {{ range .Alerts.Resolved }} {{ template "myalert" .}} {{ end }}
+        {{ end }}
+        {{ end }}
+        ```
+        * There are two sections to the notification template: 
+            1. The `myalert` template creates a single alert notification based on a specific alert.
+            2. The `mymessage` template will find all of the grouped alerts that are firing and send them in a single notification.
```
Comment (Eve832) [inline docs/sources/tutorials/create-alerts-with-logs/index.md:162]:
```suggestion
            1. The `mymessage` template will find all of the grouped alerts that are firing and send them in a single notification.
```

File: docs/sources/tutorials/create-alerts-with-logs/index.md [hunk 4]
```diff
@@ -0,0 +1,394 @@
...
+1. Verify that the logs are showing up in Grafana’s Explore view:
+
+    * Navigate to explore in Grafana.
+    * Select the Loki datasource from the drop-down.
+    * Check the toggle for **builder | code** in the top right corner of the query box and switch the query mode to builder if it’s not already selected.
+    * Select the filename label from the drop-down and choose your `web_requests.log` file from the value drop-down.
+    * Click **Run Query**.
+    * You should see logs and a graph of log volume.
+
+#### Troubleshooting the script
```
Comment (Eve832) [inline docs/sources/tutorials/create-alerts-with-logs/index.md:255]: I think you want this to be either an H3 or H2, depending on whether it should be nested under `## Optional: Use promtail with a sample log-generating script` or on the same level, respectfully

File: docs/sources/tutorials/create-alerts-with-logs/index.md [hunk 5]
```diff
@@ -0,0 +1,394 @@
...
+* If the file is empty, check that you followed the steps above to create the file and change the permissions.
+* If the file exists, verify that promtail is running and check that it is configured correctly.
+* In Grafana Explore, check that the time range is only for the last 5 minutes.
+
+## Optional: Use Docker compose to create the tutorial environment 
+
+These optional steps walk you through installing Grafana, Loki and Promtail with Docker compose. You'll also configure a log-generating script
+that generates the sample logs used in this tutorial to create alerts.
+
+#### Pre-requisites
```
Comment (Eve832) [inline docs/sources/tutorials/create-alerts-with-logs/index.md:269]: If you want this nested under the `## Optional...` heading above, make it an H3 (not H4)

**Review by Eve832 — COMMENTED** [review]
This looks really good, @mellieA , thank you and congratulations on figuring out all the things!!! 
I left some feedback and suggestions; your numbering might be askew; see my comment between lines 67 and 68.

**Comment by mellieA** [comment]
@Eve832 can you review for the changes I made?

**Review by Eve832 — APPROVED** [review]
lgtm!

**Pull request merged**